src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="main">
+    <div class="calendar calendar--month-length calendar--start-day">
```

Comment:
You are not using the modificator. Take a look at the check list. You should be able to change the number of displayed days with help of `calendar--month-length-#` i.e. `calendar--month-length-29.

Same thing with start day. You should set proper margin-left basing on the value passed there.

Reply:
Okay, I can see that it's working after I add this directly into html. But you can add some default value to your html